sharktank/setup.py

```diff
@@ -99,7 +99,6 @@ def initialize_options(self):
     extras_require={
         "testing": [
             f"pytest{get_version_spec('pytest')}",
-            f"pytest-xdist{get_version_spec('pytest-xdist')}",
```

Comment:
`pytest-xdist` isn't used, as `pytest -n` is only called in the CI. Thus I would rather go with the minimal requirements instead of pulling in non-strictly required packages.